cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""This module tests Nuage EMS events."""
+import pytest
+from wrapanapi.utils.random import random_name
```

Comment:
please sort imports according to our guidelines -  https://cfme-tests.readthedocs.io/en/master/guides/dev_guide.html